src/ir/auth_logic/ast_equality_test.cc

```diff
@@ -0,0 +1,64 @@
+//-----------------------------------------------------------------------------
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//----------------------------------------------------------------------------
+
+#include "src/common/testing/gtest.h"
+#include "src/ir/auth_logic/ast.h"
+
+namespace raksha::ir::auth_logic {
+
+TEST(AstEqualityTestSuite, Test1) {
```

Comment:
Change test name to `PredicateTest` and also indicate what is testing. e.g., 
`TEST(PredicateTest, EqualReturnsForIdenticalPredicates)`

Similarly for other TESTs.

Reply:
fixed